src/Stratis.Bitcoin/Consensus/BlockDownloadRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Consensus
+{
+    /// <summary>
+    /// A request that holds information of blocks to download.
+    /// </summary>
+    public class BlockDownloadRequest
```

Comment:
Why not internal?

Reply:
Don't know, the original was public :) Agreed for internal